sources/INSTALL.md

```diff
@@ -49,12 +66,14 @@ The steps below will build and install additional plugins (extension
 
   1. Compile with "make fm-simplex"
 
-  2. Install with "make install"
+  2. The Fm-Simplex plugin is currently built and installed
+  at the same time as the alt-ergo binary.
 
 #### The profiler plugin
 
 This plugin has been "inlined" in Alt-Ergo sources.
 
+
```

Comment:
add some text for the Why3 parser plugin (which should not be put in alt-ergo-parsers package due to licensing considerations)